coding/coding_tests/reader_writer_ops_test.cpp

```diff
@@ -39,11 +40,11 @@ namespace
   {
     FileWriter writer(fName);
 
-    srand(666);
+    base::UniformRandom<int8_t> rand;
```

Comment:
@vng fyi https://en.cppreference.com/w/cpp/numeric/random/uniform_int_distribution
> The effect is undefined if this is not one of short, int, long, long long, unsigned short, unsigned int, unsigned long, or unsigned long long.

Reply:
@Osyotr was it highlighted by VS?

Reply:
There's a `static_assert` for that in MSVC STL.
https://github.com/microsoft/STL/blob/a357ff1750d3f6dffd54b10d537e93e0accfcc92/stl/inc/random#L2340